src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -434,6 +434,24 @@ class BaseOptionsSelector extends Component {
                 shouldPreventDefaultFocusOnSelectRow={this.props.shouldPreventDefaultFocusOnSelectRow}
             />
         );
+
+        const {shouldAllowScrollingChildren} = this.props;
+        function ScrollWrapper({children}) {
+            if (!shouldAllowScrollingChildren) {
+                return children;
+            }
+            return (
+                <ScrollView contentContainerStyle={[styles.flexGrow1]}>
+                    <ScrollView
+                        horizontal
+                        contentContainerStyle={[styles.flex1, styles.flexColumn]}
+                    >
+                        {children}
+                    </ScrollView>
+                </ScrollView>
+            );
+        }
+
```

Comment:
Can we move this out of component lifecycle?

Reply:
Not sure where, what do you suggest? And what's the issues if we keep it here? thanks!

Reply:
I was thinking of static function but I think it's better to make separate component in `/components`.
I see double scroll view was used in several places so it can be reused once we make it.

Reply:
Ended up removing the component all together, let me know what you think!

Reply:
Fine to me. I found only one occurrence in AddressSearch
https://github.com/Expensify/App/blob/fe282b45cb13e01519282ccc023e5bfbd7714158/src/components/AddressSearch/index.js#L266-L273